app/configuration/Configuration.py

```diff
@@ -80,6 +85,12 @@ def override_configuration_for_hassio(self):
                     if TYDOM_PASSWORD in data and data[TYDOM_PASSWORD] != '':
                         self.tydom_password = data[TYDOM_PASSWORD]
 
+                    if DELTADORE_LOGIN in data and data[DELTADORE_LOGIN] != '':
+                        self.deltadore_login = data[DELTADORE_LOGIN]
+
+                    if DELTADORE_PASSWORD in data and data[DELTADORE_PASSWORD] != '':
+                        self.deltadore_password = data[DELTADORE_PASSWORD]
```

Comment:
The `deltadore_password` prop is printed clear text on logs; it should be masked.
 (see around lines 133-135)

It should look like
```
configuration_to_print.deltadore_password = Configuration.mask_value(configuration_to_print.deltadore_password )
```